Hunting Queries/W3CIISLog/ClientIPwithManyUserAgents.yaml

```diff
@@ -1,6 +1,8 @@
 id: 4edbb420-2df7-4089-9906-c335f065803e
 name: Same IP address with multiple csUserAgent
 description: |
+  'Alerts when same client IP connects with 1-15 different useragent strings in less than 1 hour. Limited to 50 or less connections. May indicate malicious activity. References: IIS status code mapping and Win32 Status code mapping.'
```

Comment:
No need of reference part here, rather extend the description itself.

Reply:
Removed reference string and updated description.